web-service/app.py

```diff
@@ -1,33 +1,19 @@
 import json
-from flask import Flask, render_template
+from os import remove
```

Comment:
This commit ("Webservice: Refactor app to add generate/reset buttons" is doing a bunch of unrelated refactoring work (making a PERSISTENCE_FILE variable for example). Consider splitting up.